api/src/transactions.rs

```diff
@@ -135,6 +140,7 @@ pub fn post_signing_message(
         .and(warp::body::bytes())
         .and(context.filter())
         .and_then(handle_post_signing_message)
+        .with(metrics("create_signing_message"))
```

Comment:
`post_signing_message`?  Either that or should probably rename the function `create_signing_message`